### DISCUSSION TITLE
[Feature Request] Tab trees in vertical tabs list

### DISCUSSION THREAD

**Discussion by wiggleforlife** [Floorp-Projects/Floorp#409] [Feature Request]
### Is your feature request related to a problem?

I find tab trees very helpful for my organisation - having over two hundred tabs makes finding tabs difficult without groups. 

### Describe the solution you'd like

I'd like the implementation of this to be similar to [Sidebery's](https://addons.mozilla.org/en-US/firefox/addon/sidebery/), as this is what I currently use, but I understand if there has to be differences.
Tab trees are different from Floorp's Workspaces because they are smaller groups inside of a Workspace. 
* You can fold a tree by clicking on the parent tab's icon and create a tree by dragging and dropping a tab onto another. 
* Trees can also be created by opening a link in a new tab - the original tab is the parent tab and the new tab becomes a child tab.
* Trees can be inside of other trees

### Additional context

![image](https://github.com/Floorp-Projects/Floorp/assets/46230102/971a8e91-87b5-4a0e-9476-85ad4b4bf15a)
A tree with more trees as children.
![image](https://github.com/Floorp-Projects/Floorp/assets/46230102/8e486a1f-9f7b-46d3-b555-64b717d92103)
A folded tree.
![image](https://github.com/Floorp-Projects/Floorp/assets/46230102/c4d630b9-9e84-4c56-82aa-c96e4196a572)
The arrow that appears to indicate a tab tree being created while dragging and dropping.

Love what you're doing with this browser, especially since it's using Gecko and Firefox instead of Chromium!! Thank you!!

**Comment by Doodah65** [comment]
I love the Floorp's speed and privacy features but without this feature it does not really support my workflow. Using Sideberry in Floorp is also not a good solution for me.

**Comment by MurzNN** [comment]
I prefer tab trees too, and use a wonderful extension https://addons.mozilla.org/en-US/firefox/addon/tree-style-tab/ that works pretty good in Floorp! To simplify identifying concrete websites I even created an extension to colorize tabs by domain: https://addons.mozilla.org/en-US/firefox/addon/tst-colored-tabs/ ;)

But native implementation should be much better for performance, so, adding +1 for this ;)
